tensorboard/webapp/metrics/views/main_view/common_selectors.ts

```diff
@@ -292,12 +286,8 @@ export const getAllPotentialColumnsForCard = memoize((cardId: string) => {
   );
 });
 
-export const factories = {
```

Comment:
This was being used to mock selector factories in tests. Now there aren't any so this is unneeded.